go-apps/meep-metrics-engine/api/swagger-v2.yaml

```diff
@@ -0,0 +1,790 @@
+      values:
+      - loss: 0.003
+        time: "2019-11-24T12:45:00-5:00"
+        tput: 100000.0
+        lat: 5.123
+      - loss: 0.003
+        time: "2019-11-24T12:45:00-5:00"
+        tput: 100000.0
+        lat: 5.123
+      name: "network metrics"
+  EventQueryParams:
+    type: "object"
+    properties:
+      tags:
+        type: "array"
+        description: "Tag array<br>combination of: target, type"
+        items:
+          $ref: "#/definitions/Tag"
+      fields:
+        type: "array"
+        description: "Field array<br>combination of: lat, tput, loss"
+        items:
+          type: "string"
+          example: "event"
+          description: "Queried value"
+      scope:
+        $ref: "#/definitions/Scope"
+    description: "Event metrics query parameters"
+    example:
+      scope:
+        duration: "10s"
+        limit: 60
+      fields:
+      - "event"
+      tags:
+      - name: "type"
+        value: "MOBILITY"
+  EventValue:
+    type: "object"
+    properties:
+      time:
+        type: "string"
+        example: "2019-11-24T12:45:00-5:00"
+        description: "Time of event value"
+      description:
+        type: "string"
+        example: "Mobility event ue1 to poa3"
+        description: "Event description"
+    description: "Value of a single event metric"
+    example:
+      description: "Mobility event ue1 to poa3"
+      time: "2019-11-24T12:45:00-5:00"
+  EventQueryResponse:
+    type: "object"
+    properties:
+      name:
+        type: "string"
+        example: "event metrics"
+        description: "Response name"
+      columns:
+        type: "array"
+        description: "columns included in response based on queried values"
+        items:
+          type: "string"
+          example: "event"
+          description: "Queried value"
+      values:
+        type: "array"
+        items:
+          $ref: "#/definitions/EventValue"
+    description: "Event metrics query response"
+    example:
+      columns:
+      - "event"
+      values:
+      - description: "Mobility event ue1 to poa3"
+        time: "2019-11-24T12:45:00-5:00"
+      - description: "Mobility event ue1 to poa2"
+        time: "2019-11-24T12:44:00-5:00"
+      name: "event metrics"
+  EventSubscriptionParams:
+    type: "object"
+    properties:
+      clientCorrelator:
+        type: "string"
+        example: "83"
+        description: "Uniquely identifies this create subscription request. If there\
+          \ is a communication failure during the request, using the same clientCorrelator\
+          \ when retrying the request allows the operator to avoid creating a duplicate\
+          \ subscription."
+      callbackReference:
+        $ref: "#/definitions/Events_callbackReference"
+      eventQueryParams:
+        $ref: "#/definitions/EventQueryParams"
+      recurrence:
+        type: "integer"
+        example: 1
+        description: "Recurrence of recurring-time-based notifications"
```

Comment:
Rename property: updateInterval

Description: "Notification interval in seconds, disabled if set to 0"

Reply:
fixed